{"name": "Логин курьера. Попытка логина несуществующего курьера. Возвращает верный статус-код", "status": "passed", "description": "тест ручки \"Логин курьера\" ", "start": 1724853579915, "stop": 1724853579966, "uuid": "31788c40-5493-44d8-b01a-846f49a5023f", "historyId": "bde7c88a6c0554edb2f71305ad68b20e", "testCaseId": "bde7c88a6c0554edb2f71305ad68b20e", "fullName": "tests.TestCourierLogin#test_courier_not_existing_login_returns_correct_status_code", "labels": [{"name": "suite", "value": "tests"}, {"name": "subSuite", "value": "TestCourierLogin"}, {"name": "host", "value": "FanatovPC"}, {"name": "thread", "value": "31284-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests"}]}